account_payment_partner/models/account_invoice.py

```diff
@@ -32,11 +32,7 @@ def _onchange_partner_id(self):
                     self.partner_bank_id =\
                         self.commercial_partner_id.bank_ids[0]
             elif self.type == 'out_invoice':
-                pay_mode = self.partner_id.customer_payment_mode_id
-                self.payment_mode_id = pay_mode
-                if pay_mode and pay_mode.bank_account_link == 'fixed':
-                    self.partner_bank_id = pay_mode.fixed_journal_id.\
-                        bank_account_id
+                self.payment_mode_id = self.partner_id.customer_payment_mode_id
```

Comment:
@alexis-via ¿Can you review?
I remove this because it causes a problem if you have a direct debit payment method with bank_account_link == 'fixed'
I have contemplated the original functionality and I have also created the migration script to avoid causing problems